sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,10 +2,22 @@
 
 ## 5.3.0b2 (Unreleased)
 
+This version of the client library defaults to the service API version `2022-10-01-preview`.
```

Comment:
We already mentioned the version updates in the previous beta. Should we keep mentioning it until it changes?

Reply:
I think it's useful information. I feel like we usually bump the API version when we do a release and this time we're not so it's worth calling out.